solarforecastarbiter/metrics/calculator.py

```diff
@@ -503,15 +487,17 @@ def _transform_prob_forecast_value_and_prob(proc_fx_obs,
         Forecast physical values and forecast probabilities.
     """
     fx_fx_prob = []
-    df = getattr(proc_fx_obs, attr)
-    assert isinstance(df, pd.DataFrame)
-    for col in df.columns:
+    data = getattr(proc_fx_obs, attr)
+    if isinstance(data, pd.Series):
```

Comment:
comment on when this is needed (I believe for `ProbabilisticForecastConstantValue` data).

Reply:
added